test/arithmetic/floating_point/floating_point_conversion_test.dart

```diff
@@ -14,9 +14,13 @@ import 'package:test/test.dart';
 
```

Comment:
do converters, multipliers, etc. that don't support DAZ/FTZ need to throw an exception?